src/Bicep.Core.RegistryClient/Bicep.Core.RegistryClient.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <Nullable>annotations</Nullable>
```

Comment:
>annotations [](http://example.com/codeflow?start=14&length=11)

This new project was generated by autorest and is not following our standard settings. I'm expecting that the code generator has built-in assumptions on these project settings in order for the generated code to compile, so I'm leaving them alone.